ast/cloner.go

```diff
@@ -65,9 +65,6 @@ func copyStruct(in Node) (out Node) {
 	case *DefaultExpr:
 		nv := *v
 		out = &nv
-	case *IdentifierExpr:
```

Comment:
Because it is replaced by `ColumnNameExpr`, for "CURRENT TIMESTAMP", it is replaced by "FuncCallExpr".